config/jobs/kubernetes/sig-scalability/sig-scalability-presubmit-jobs.yaml

```diff
@@ -489,3 +489,55 @@ presubmits:
             requests:
               cpu: 1
               memory: "2Gi"
+
+  - name: pull-perf-tests-unit-tests
+    decorate: true
+    always_run: true
+    spec:
+      containers:
+      - image: golang:1:13
+        command:
+        - make
+        args:
+        - test
```

Comment:
Hmm - I can't see Makefile in perf-tests repo.

Reply:
There is another PR for perf-tests repo I've just send, please see it.

Reply:
Yeah - just saw that. Sorry. Commented on that.

Reply:
https://github.com/kubernetes/perf-tests/pull/1216